client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -41,6 +45,35 @@ const Settings = ( props: SettingsPageProps ): JSX.Element => {
 		return <Breadcrumbs items={ items } mobileItem={ mobileItem } />;
 	};
 
+	const getContactInformationAccordion = () => {
+		const { whoisData } = props;
+		const { privateDomain } = domain;
+
+		const contactInformation = findRegistrantWhois( whoisData );
+
+		if ( ! contactInformation ) {
+			props.requestWhois( props.selectedDomainName );
+			return null;
+		}
+
+		const contactInfoFullName = `${ contactInformation.fname } ${ contactInformation.lname }`;
+		const privacyProtectionLabel = privateDomain
+			? translate( 'privacy protection on' )
+			: translate( 'privacy protection off' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Privacy protection: off' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-management-contact-information)